src/MOCKernel.h

```diff
@@ -0,0 +1,121 @@
+/**
+ * @file MOCKernel.h
+ * @brief An MOCKernel object
+ * @date May 5, 2015
+ * @author Geoffrey Gunow, MIT, Course 22 (geogunow@mit.edu)
+ */
+
+#ifndef MOCKERNEL_H_
+#define MOCKERNEL_H_
+
+#include "Python.h"
```

Comment:
This include of "Python.h" should be wrapped with an `#ifdef` for the `SWIG` macro as so:

```c++
#ifdef SWIG
#include "Python.h"
#endif
```

This is how we include this file elsewhere. There is a very good reason for doing this, though it slips my mind at this time at night.